keyboards/wolf/frog_np/readme.md

```diff
@@ -0,0 +1,13 @@
+# Frog NP
+
+The following is the QMK Firmware for the Frog Numpad PCB.
+
+---
+
+* Keyboard Maintainer: ToastyStoemp
+
+Make example for this keyboard (after setting up your build environment):
+
+    make wolf/Frog NP:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```